docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+
+| Label  | Purpose |
+| ------------- | ------------- |
+| work in progress | The assignee is still making changes to this branch |
+| needs review | The work is ready for testing and reviewers are assigned |
+| review in progress | Indicates the review process is underway |
+| pass code review | All changes follow coding standards and best practices |
+| pass functional review | The work functions as described in the JIRA ticket |
+| ready to merge | The changes are approved and ready to be merged |
+| needs work | The PR did not pass review and feedback was provided |
+| needs help | Use if help is needed; leaving details in the comments |
+| don't merge | Do not merge the work; explanation in the comments |
+
+### Best practices
+
+- Generally, pull requests should resolve a single Jira ticket. Try to avoid combining multiple tickets into a single pull request. There may be instances where it makes sense to do otherwise but please use discretion.
+- Avoid long-standing PRs as the work can become a bottleneck or create future merge conflicts. Regularly ping reviewers in Teams and JIRA to ensure the review process can move swiftly.
+- Try to keep pull requests reasonably small and discrete. If you are beginning to work on a story and it feels like it will result in a giant pull request with lots of custom code, changes across many features, and lots of testing scenarios, think about how you might break down the story into smaller subtasks that could be incrementally developed and tested. Create subtasks or potentially even new stories within JIRA. If you are unsure about how or are unable to do this, please reach out to the project Product Owner.
+
+## Code standards
+
+Development standards will be rigorously enforced on this project. Drupal maintains coding standards that apply to the core and contributed code. This is the basis for code linting and should be considered a reference when there are uncertainties. While linting is required, the following resources are more aspirational:
+
+- [Drupal Coding standards](https://www.drupal.org/docs/develop/standards)
+- [Twig coding standards](https://www.drupal.org/docs/develop/coding-standards/twig-coding-standards)
+- [JavaScript coding stand](https://www.drupal.org/docs/develop/standards/javascript)
+- [CSS Architecture](https://www.drupal.org/docs/develop/standards/css/css-architecture-for-drupal-8)
+
+### Code linting
+
+Linting tools help avoid errors and ensure code is formatted in a way that is consistent between all contributors. It is highly encouraged to check code standards over the codebase before committing. NPM scripts are included in this project to help check your custom code for common errors. From within the project, you can run:
+
+- `npm run lint` to lint all of your custom code.
+- `npm run lint-php` to lint only custom PHP code.
+- `npm run lint-js` to lint only custom Javascript code.
```

Comment:
I'm not able to run any of these. Possible missing dependencies and scripts?

```
$ npm run lint

> lint
> ./scripts/node/lint.sh

  [Symfony\Component\Console\Exception\CommandNotFoundException]
  Command "lint" is not defined.

$ npm run lint-php
npm ERR! Missing script: "lint-php"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vincentmassaro/.npm/_logs/2022-04-04T21_02_53_394Z-debug.log

$ npm run lint-js                                                                                                                                                                                                                                                                                 
npm ERR! Missing script: "lint-js"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vincentmassaro/.npm/_logs/2022-04-04T21_02_57_611Z-debug.log

$ npm run                                                                                                                                                                                                                                                                                         
Scripts available in yalesites-upstream via `npm run-script`:
  confim
    ./scripts/node/confim.sh
  confex
    ./scripts/node/confex.sh
  import-local-db
    ./scripts/node/import-data.sh
  build
    ./scripts/node/build.sh
  rebuild
    ./scripts/node/rebuild.sh
  export-db
    ./scripts/node/export-db.sh
  get-db
    ./scripts/node/get-db.sh
  get-files
    ./scripts/node/get-files.sh
  get-assets
    ./scripts/node/get-files.sh
  theme
    ./scripts/node/theme.sh
  build-theme
    ./scripts/node/build-theme.sh
  lint
    ./scripts/node/lint.sh
```

Reply:
I got a little ahead of myself when writing this documentation. These linting tools are in the yalesites_profile repo, not this project repo. I will remove these for now and add a ticket to add linting to the project repo. Ideally pull that task in this sprint.